cloud2cloud-gateway/service/retrieveAllDevicesSubscription_test.go

```diff
@@ -101,7 +101,7 @@ func TestRequestHandlerRetrieveDevicesSubscription(t *testing.T) {
 			},
 			wantCode:        http.StatusUnauthorized,
 			wantContentType: textPlain,
-			want:            "invalid token: could not parse token: token has invalid claims: inaccessible uri",
+			want:            "unknown issuer",
```

Comment:
we lose the real error, which remains `inaccessbile uri` from `DeniedClaims`. We should at least make an error log in DeniedClaims with the error is created, so we know that this occured. 

`func (c *MultiKeyCache) GetOrFetchKeyWithContext` gets called before `func (c DeniedClaims) Validate()` by the external jwt/v5 package. I guess we could try casting the token.Claims in `GetOrFetchKeyWithContext` to DeniedClaims and return the error if it's not nil at the begining of the function. Though this is a maybe a bit of an overkill, the error log would be sufficient for me.

Reply:
fixed -> if is Error interface then it will return directly the error

Reply:
👍